python/glow/wgr/functions.py

```diff
@@ -128,3 +129,38 @@ def block_variants_and_samples(variant_df: DataFrame, sample_ids: List[str],
     output = blocked_gt, index_map
     assert check_return_type(output)
     return output
+
+
+def pivot_for_gwas(spark: SparkSession, label_df: pd.DataFrame) -> DataFrame:
+    """
+    Pivots a Pandas DataFrame into a Spark DataFrame with a convenient format for Glow's GWAS
```

Comment:
I'm a little iffy about the term "pivot" here; generally, a pivot table contains summary statistics. I know Pandas uses `pivot` when reshaping though.